src/main/java/version1/gameUtil/settings/UIConfigurations.java

```diff
@@ -158,6 +158,7 @@ public static void configure(ModeScreen modeScreen){
                 .createMediumButton(mediumButtonText)
                 .createHardButton(hardButtonText)
                 .createGoToMenuButton(goToMenuText)
+                .createGoToMediumMazeButton(mediumButtonText)
```

Comment:
Lines 158 and 161 seem to be doing the same task.  You did not have to create new methods, you just had to fill in the ones that were already there. 

It is my bad, I should have mentioned that yesterday.  So we will need to remove one of them and keep the other. Same thing for all the other screens.